source/plugins/ruby/in_containerinventory.rb

```diff
@@ -82,6 +83,15 @@ def enumerate
                   if hostName.empty? && !containerRecord["Computer"].empty?
                     hostName = containerRecord["Computer"]
                   end
+                  if addonTokenAdapterImageTag.empty? && ExtensionUtils.isAADMSIAuthMode()
+                     if !containerRecord["ElementName"].nil? && !containerRecord["ElementName"].empty? &&
+                      containerRecord["ElementName"].include?("kube-system") &&
```

Comment:
so if i have a kube-system-2 or my-kube-system namespace with a pod having same container name, we will pick that image ?